aws-s3-transfer-manager/examples/cp.rs

```diff
@@ -252,8 +252,7 @@ async fn do_upload(args: Args) -> Result<(), BoxError> {
         .bucket(bucket)
         .key(key)
         .body(stream)
-        .send()
-        .await?;
+        .initiate()?;
```

Comment:
Not really, Aaron suggested changing the name from `send` to avoid confusion with the Rust SDK at https://github.com/awslabs/aws-s3-transfer-manager-rs/pull/73#discussion_r1838162739, and I couldn't think of something better than initiate/send.